README.md

```diff
@@ -18,6 +18,7 @@ const myAppOptions = {
   ...APIOptions,
   clientSecret: 'my-client-secret'
   clientId: 'my-super-secret-app-id',
+  withEmailScope: false, // only supported for non-embedded apps
```

Comment:
would it be better to just be able to override `scopes` wholesale so changes like this aren't needed in the future?